homeassistant/components/group/strings.json

```diff
@@ -190,5 +190,55 @@
         "product": "Product"
       }
     }
+  },
+  "services": {
+    "reload": {
+      "name": "Reload",
+      "description": "Reloads group configuration, entities, and notify services from YAML-configuration."
+    },
+    "set": {
+      "name": "Set",
+      "description": "Creates/Updates a user group.",
+      "fields": {
+        "object_id": {
+          "name": "Object ID",
+          "description": "Group ID and part of entity ID."
```

Comment:
```suggestion
          "description": "Combination of the group ID and part of the entity ID."
```
not sure about this

Reply:
It is not a combination. It is a bit off and worded odd too.
So an entity ID consists out of two parts: an domain and an object ID.

Entity ID format: `<domain>.<object_id>`
Example domain: `group`
Example object ID: `window_lights`
Example entity ID: `group.window_lights`